frontend/src/main/webapp/ViewTrip.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Trip Me Out</title>
+    <script src="ViewTripJavaScript.js"></script>
+  </head>
+  <body onload="setTripPlaceContainerWithServlet()">
+    <h1>Trip Me Out!</h1>
+    <h2> View Trip</h2>
+    <button>Fake Plus Button</button>
+    <button>Fake Minus Button</button>
+    <section id="Trip-Place">
+        <p>Trips will live below</p>
+        <dl id="places"></dl>    
```

Comment:
I'm not sure how empty `<dl>` elements render. Consider sticking this inside of a container div and creating the `<dt>` on the fly as part of the render process if this adds weirdness to the page. If the empty `<dt>` doesn't display strangely, this is fine to keep as is.

Reply:
I only used a `<dl>` because it was a description list but I will try using a div container and see if it still works.